tests/mkcw.bats

```diff
@@ -58,7 +58,7 @@ function mkcw_check_image() {
     skip "cryptsetup not found"
   fi
   _prefetch busybox
-  _prefetch bash
+  _prefetch $SAFEIMAGE
```

Comment:
If there is/was anything special about `bash`, it was not obvious from comments, so it gets nuked

Reply:
The `bash` image has three layers, compared to, say `busybox`, which has one.  The test attempts to confirm that content from the base image and content added at conversion time both show up in the disk image that `mkcw` is generating.

Reply:
Thank you! `$SAFEIMAGE` has two layers, would that be sufficient? If not, could you suggest a three-layer image from this list? https://github.com/containers/automation_images/pull/357/files#diff-7b013edc1f5a127319d75ae1f8ae443e0bfa4d891960c27ee284bbb107ddd290R56

Reply:
One of those layers is empty, but it should be fine.  The error we were attempting to avoid was causing the disk image to lose content from layers that weren't the last one (and as far as this test is concerned, for images with one layer, the only layer is also the last layer), and the presence of the second layer, empty as it is, means the image still meets the test expectations.

Reply:
Thanks. Repushed with a warning comment